src/alert/styles.scss

```diff
@@ -25,9 +25,14 @@
   position: relative;
   display: grid;
   grid-template-columns: 1fr;
-  border-radius: awsui.$border-radius-alert;
-  border: awsui.$border-field-width solid;
-  padding: awsui.$space-alert-vertical awsui.$space-alert-horizontal;
+  border-start-start-radius: awsui.$border-radius-alert;
+  border-start-end-radius: awsui.$border-radius-alert;
+  border-end-start-radius: awsui.$border-radius-alert;
+  border-end-end-radius: awsui.$border-radius-alert;
```

Comment:
I'm not sure if the "no shorthands" rule makes a ton of sense for `border-radius`. It's almost always the same value for all.

Reply:
We discussed it on the previous PR: https://github.com/cloudscape-design/components/pull/1729#discussion_r1391666829

`border-radius` is a positional property: `[top] [right] [bottom] [left]`. For safety and ease of coding we decided to ban all cases rather than analysing possible exceptions.

Open for discussion (we can write a custom stylelint rule), but I would not block the ongoing work because of that

Reply:
I would imagine it would be as simple as removing "border" from this line: `"property-disallowed-list": ["border", "border-radius", "border-style", "margin", "padding"],`

Reply:
True, for `border` the CSS grammar does not allow setting more than one value. So it is all-way-round anyway.

But the `border-radius` will have to stay expanded. For 4 `border:` occurrences in this PR we can revert them back

cc @DaemonCahill

Reply:
Good callout! I will update the linting rules to keep the `border` shorthand but still block `border-radius`.